src/calculation/Calculator.ts

```diff
@@ -532,6 +533,9 @@ export async function calculateLibraryDataRequirements(
   libraryBundle: fhir4.Bundle,
   options: CalculationOptions = {}
 ): Promise<DRCalculationOutput> {
+  // omit measurementPeriodStart/measurementPeriodEnd since there is no measure
```

Comment:
I think this is good and we don't need to throw an error, but should we have a warning that the measurement period is omitted? That way a user isn't confused by the output? A warning might be unnecessary so in that case maybe just adding something small to the README that addresses this.

Reply:
Yeah I agree. I think best course would just be to add a note to the documentation (and maybe the cli descriptions as well) that says measurementPeriodStart and measurementPeriodEnd are ignored for libraryDataRequirements

Reply:
Good call - I think we should definitely add something to the documentation. Thoughts on adding a `console.warn` as well or do we think that's unnecessary?

Reply:
I think that's probably unnecessary. We do use `console.warn` in fqm-execution, but not widely, and I think a documentation update is enough info here.